.gitignore

```diff
@@ -1,38 +1,41 @@
-Debug/
-PageSpeed/lib/Debug/
-Release/
-lib-release/
-lib-debug/
-wptdriver tests/
-/ipch/
-.idea/
-.project
-*.vcxproj.user
-*.vcproj.*.user
-*~
-*.pyc
-*.testsettings
-*.aps
-webpagetest.opensdf
-webpagetest.sdf
-*.suo
-webpagetest.vsmdi
-.svn
-*.last
-.DS_store
-*.ppw
-*.ppd
-*.ppj
-*.ppx
-*.dsk
-*.idx
-*.ncb
-*.$wrk
-agent/js/*.log
-agent/js/node_modules/.bin/jshint
-agent/js/node_modules/jshint/
-agent/js/results/
-agent/js/src-cov/
-agent/js/runtmp_*
-.vagrant
-/www/installers/browsers/nightly.dat
+Debug/
+PageSpeed/lib/Debug/
+Release/
+lib-release/
+lib-debug/
+wptdriver tests/
+/ipch/
+.idea/
+*.vcxproj.user
+*.vcproj.*.user
+*~
+*.pyc
+*.testsettings
+*.aps
+webpagetest.opensdf
+webpagetest.sdf
+*.suo
+webpagetest.vsmdi
+.svn
+*.last
+.DS_store
+*.ppw
+*.ppd
+*.ppj
+*.ppx
+*.dsk
+*.idx
+*.ncb
+*.$wrk
+agent/js/*.log
+agent/js/node_modules/.bin/jshint
+agent/js/node_modules/jshint/
+agent/js/results/
+agent/js/src-cov/
+agent/js/runtmp_*
+.vagrant
+/www/installers/browsers/nightly.dat
+.gradle
+*/webdriver-support/java/build/
+*.iml
+logs/

```

Comment:
This is problem a line-ending problem. See if you can remove this from the commit